packages/shell-api/src/cursor.spec.ts

```diff
@@ -54,8 +54,8 @@ describe('Cursor', () => {
       expect((await toShellResult(cursor.help)).type).to.equal('Help');
     });
 
-    it('returns the same cursor', () => {
-      expect(cursor.map()).to.equal(cursor);
+    it('map() returns a new cursor', () => {
```

Comment:
aren't all the tests about `map()`?

Reply:
I’m not sure if I’m understanding you correctly… some of the tests here are about `cursor.map()`, yes, but not all of them

Reply:
i've seen it now, i thought we were doing `describe('method') .. it('does this and that')` everywhere, but it looks like in that file is different.